lib/algos/list/skip.js

```diff
@@ -0,0 +1,88 @@
+const assert = require('assert');
+
+const { FILTER_END, FILTER_SKIP, SKIP_NONE } = require('./tools');
+
+
+const MAX_STREAK_LENGTH = 100;
+
+/**
+ * Handle the filtering and the skip mechanism of a listing result.
+ */
+class Skip {
+    /**
+     * @param {Object} params    - skip parameters
+     * @param {Object} extension - delimiter extension used (required)
```

Comment:
This looks like additional arguments to the class. Can you update to make it apparent they are properties of `params`?

Reply:
done